### ISSUE TITLE
[Bug Report] Able to select dates past maxDate.

### ISSUE THREAD

**Issue by rwithik** [onesine/react-tailwindcss-datepicker#50]
If the user clicks the next month's dates, they are able to select dates past the `maxDate`. I assume this will work similarly for `disabledDates` and `minDate`. Attaching a video of the same and the props passed to the datepicker.

https://user-images.githubusercontent.com/34149531/214136462-104f2bf6-ebf6-49ef-baad-06f62314a37f.mov

```
<Datepicker
      containerClassName={"w-1/5 bg-white font-normal self-end z-10"}
      inputClassName="rounded-md"
      primaryColor="blue"
      displayFormat="MMM, DD YYYY"
      separator="-"
      showShortcuts={true}
      placeholder="Select the date range"
      maxDate={new Date()}
      useRange={false}
      readOnly
      value={dateRange}
      onChange={onChange}
    />
```

**Comment by JefteCaro** [comment]
Hi @rwithik 

Thank you for opening this issue.

This will be fixed in the next release.

**Comment by JefteCaro** [comment]
Fix in [`v1.4.1`](https://github.com/onesine/react-tailwindcss-datepicker/releases/tag/v1.4.1)

Closing this issue
